07LessonSeven/homework/css/style.css

```diff
@@ -0,0 +1,57 @@
+body {
+  font-family: arial, sans-serif;
+}
+
+header {
+  font-size: 25px;
+  font-weight: 200;
+  margin-left: 70px;
+}
+
```

Comment:
we want the header, nav, and green box to line up vertically, so make them all the same % padding